armi/materials/material.py

```diff
@@ -845,9 +856,36 @@ class FuelMaterial(Material):
     class1_wt_frac = None
     class1_custom_isotopics = None
     class2_custom_isotopics = None
-    puFrac = 0.0
-    uFrac = 0.0
-    zrFrac = 0.0
+    puFracDefault = 0.0
+    uFracDefault = 0.0
+    zrFracDefault = 0.0
+
+    def __init__(self):
+        self._puFrac = 0
```

Comment:
We will still need attributes for `uFrac` and `zrFrac` -- see `duplicate` method, so we can have `setters` and `getters` for accessing to keep `self._uFrac`, `self._zrFrac`, and `self._puFrac` private.

Reply:
code works as is, so we dont need them immediately without clear expectations on behavior of u and zr frac with depletion